### PR TITLE
Exit if no indexes exist

### PR DESCRIPTION
**Pull request by toddself**
Fixes: https://github.com/yoshuawuyts/choo-log/issues/57

### CODE REVIEW

**Comment by toddself** [comment]
image

File: index.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ function logger (opts) {
     function renderPerformance () {
       var entries = window.performance.getEntriesByName('choo:render')
       var index = entries.length - 1
+      if (index < 0) return
```
Comment (yoshuawuyts) [inline index.js]: This should probably be `return log.info('render')` so we still log the `'render'` event - we might drop it otherwise :(

**Review by yoshuawuyts — CHANGES_REQUESTED** [review]

**Comment by s3ththompson** [comment]
Are there additional changes necessary for this PR to land?

**Review by yoshuawuyts — APPROVED** [review]

**Comment by yoshuawuyts** [comment]
`v6.1.2`

**Pull request merged**